packages/core/src/services/db/backends/ephem-db.provider.ts

```diff
@@ -18,7 +18,8 @@ export class EphemDBProvider implements BaseDBProvider {
    * @param fallback A fallback value if the key doesn't exist.
    * @returns the stored value or the fallback.
    */
-  public async get<T>(key: string, fallback?: T): Promise<T | DBResult> {
+  public async get<T>(_key: Buffer | string, fallback?: T): Promise<T | DBResult> {
```

Comment:
Hmm I'd vote to keep the parameter as `key` instead of `_key`.

Reply:
Ah so the reason I was doing this is I can't reassign the parameter & didn't feel like using a different variable name for the key that's used in the function. For instance I could use `const stringifiedKey = this.convertToString(key)` and then use the stringifiedKey.